modules/installation-guide/partials/proc_configuring-workspace-exposure-strategies-using-an-operator.adoc

```diff
@@ -13,31 +13,53 @@ link:https://docs.openshift.com/container-platform/latest/operators/olm-what-ope
 
 .Procedure
 
-When deploying {prod-short} using the Operator, configure the intended strategy by modifying the `spec.k8s.ingressStrategy` property of the CheCluster Custom Resource object YAML file.
+When deploying {prod-short} using the Operator, configure the intended strategy by modifying the `spec.server.serverExposureStrategy` and  optionally `spec.server.singleHostWorkspaceExposureType` properties of the CheCluster Custom Resource object YAML file.
 To activate changes done to CheCluster YAML file, do one of the following:
 
 * Create a new cluster by executing the `{prod-cli}` command with applying a patch. For example:
 +
 [subs="+quotes,+attributes"]
 ----
-$ {prod-cli} server:start ... --che-operator-cr-patch-yaml=patch.yaml
+$ {prod-cli} server:start  --installer=operator --platform=... --che-operator-cr-patch-yaml=patch.yaml
 ----
 
-patch.yaml should contain the following:
+patch.yaml should contain the following for `native` single-host exposure type:
```

Comment:
I think default on openshift will be `gateway`. So the patch you have here uses `native` only on Kubernetes.

Reply:
added